### PR TITLE
test(integ): delete ssm params and iam roles after executing the tests

### PR DESCRIPTION
**Pull request by bvtujo**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by huanjani — APPROVED** [review]
Thanks for doing this!

File: internal/pkg/deploy/cloudformation/cloudformation_integration_test.go [hunk 1]
```diff
@@ -16,13 +16,20 @@ import (
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/aws/session"
 	awsCF "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/aws/iam"
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/cloudformation_integration_test.go:19]: yay our wrapper that deletes the policies before the role!

File: internal/pkg/deploy/cloudformation/cloudformation_integration_test.go [hunk 2]
```diff
@@ -16,13 +16,20 @@ import (
...
 	"github.com/aws/copilot-cli/internal/pkg/aws/identity"
 	"github.com/aws/copilot-cli/internal/pkg/config"
 	"github.com/aws/copilot-cli/internal/pkg/deploy"
 	"github.com/aws/copilot-cli/internal/pkg/deploy/cloudformation"
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	fmtIAMRoleARN         = "arn:aws:sts::%s:role/%s"
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/cloudformation_integration_test.go]: huh does this need to be "iam" instead of "sts"?

Comment (bvtujo) [reply internal/pkg/deploy/cloudformation/cloudformation_integration_test.go]: lmao you're totally right, this was a dumb typo.

**Comment by bvtujo** [comment]
SSM test properly cleans up parameters. 

Waiting on the output of the fixed CFN test.

**Comment by bvtujo** [comment]
Looks like IAM roles also get appropriately deleted!

**Review by efekarakus — APPROVED** [review]

**Pull request merged**